Matrix.py

```diff
@@ -0,0 +1,181 @@
+"""Matrices calculator class.
+
+The matrices calculator can be used to add, sub, multiplication (by scalars or matrices) and scalar
+division. also you can compare matrices or using them as a dictionary key. the program only works
+with squared matrices.
+
+    Typical usage example:
+
+    x = Matrix(((1,2,3),(4,5,6),(6,7,8)))
+    y = Matrix.unity(3)
+    new_matrix = x * y
```

Comment:
redundant

Reply:
it was in the doc guide but i will delete it